.travis.yml

```diff
@@ -33,9 +34,133 @@ matrix:
 before_script:
   - export PATH="$HOME/.composer/vendor/bin:$PATH"
   - |
+    ORIG_DIR=`pwd`
     if [[ ! -z "$WP_VERSION" ]] ; then
       bash bin/install-wp-tests.sh wordpress_test root '' localhost $WP_VERSION
-      if [[ ${TRAVIS_PHP_VERSION:0:2} == "5." ]]; then
+      if [[ ${SWITCH_TO_PHP:0:3} == "5.2" ]] || [[ ${SWITCH_TO_PHP:0:3} == "5.3" ]]; then
+        # install php runtime dependencies
+        sudo apt-get install -y curl libxml2 libfreetype6 libpng12-0 libjpeg8 libgd3 libxpm4 \
+        libltdl7 openssl gettext libgettextpo0 libmhash2 libmcrypt4 mysql-client
+
+        if [[ ${SWITCH_TO_PHP:0:3} == "5.2" ]]; then
+          PHPBREW_BUILT_CHECK=$HOME/.phpbrew/php/php-5.2.17/bin/php
```

Comment:
there are includes, phpunit libs, config files etc. in the other dirs. The only one we could get rid of safely is the headers in include/ would save a couple of Mb.